de.metas.handlingunits.base/src/main/sql/postgresql/system/5493530_sys_gh4018_SYSConfig_Enforce_DDOrderline_M_HU_PI_Item_Product.sql

```diff
@@ -0,0 +1,5 @@
+-- 2018-05-14T16:22:03.594
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541210,'S',TO_TIMESTAMP('2018-05-14 16:22:03','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.handlingunits','Y','DD_OrderLine_Enforce_M_HU_PI_Item_Product',TO_TIMESTAMP('2018-05-14 16:22:03','YYYY-MM-DD HH24:MI:SS'),100,'-')
```

Comment:
* why did you add the `-` instead of Y or N? 
* please add some documentation about what the sysconfig's implications are. Both to this `AD_SysConfig` record and the issue. then flag the issue as `type:documentation`. In particular, please document how *this* sysconfig affects the actual issue around the CuQty

Reply:
Done